src/main/java/org/dependencytrack/persistence/jdbi/FindingDao.java

```diff
@@ -0,0 +1,574 @@
+                 , "COMPONENT"."PURL" AS "componentPurl"
+                 , "COMPONENT"."CPE" AS "componentCpe"
+                 , "VULNERABILITY"."UUID" AS "vulnUuid"
+                 , "VULNERABILITY"."SOURCE" AS "vulnSource"
+                 , "VULNERABILITY"."VULNID"
+                 , "VULNERABILITY"."TITLE" AS "vulnTitle"
+                 , "VULNERABILITY"."SUBTITLE" AS "vulnSubtitle"
+                 , "VULNERABILITY"."DESCRIPTION" AS "vulnDescription"
+                 , "VULNERABILITY"."RECOMMENDATION" AS "vulnRecommendation"
+                 , "VULNERABILITY"."PUBLISHED" AS "vulnPublished"
+                 , "VULNERABILITY"."SEVERITY" AS "vulnSeverity"
+                 , CAST(STRING_TO_ARRAY("VULNERABILITY"."CWES", ',') AS INT[]) AS "CWES"
+                 , "VULNERABILITY"."CVSSV2BASESCORE"
+                 , "VULNERABILITY"."CVSSV3BASESCORE"
+                 , "VULNERABILITY"."CVSSV2VECTOR"
+                 , "VULNERABILITY"."CVSSV3VECTOR"
+                 , "VULNERABILITY"."OWASPRRLIKELIHOODSCORE"
+                 , "VULNERABILITY"."OWASPRRTECHNICALIMPACTSCORE"
+                 , "VULNERABILITY"."OWASPRRBUSINESSIMPACTSCORE"
+                 , "VULNERABILITY"."OWASPRRVECTOR"
+                 , JSONB_VULN_ALIASES("VULNERABILITY"."SOURCE", "VULNERABILITY"."VULNID") AS "vulnAliasesJson"
+                 , "EPSS"."SCORE" AS "epssScore"
+                 , "EPSS"."PERCENTILE" AS "epssPercentile"
+                 , "FINDINGATTRIBUTION"."ANALYZERIDENTITY"
+                 , "FINDINGATTRIBUTION"."ATTRIBUTED_ON"
+                 , "FINDINGATTRIBUTION"."ALT_ID"
+                 , "FINDINGATTRIBUTION"."REFERENCE_URL"
+                 , "ANALYSIS"."STATE" AS "analysisState"
+                 , "ANALYSIS"."SUPPRESSED"
+              FROM "COMPONENT"
+             INNER JOIN "COMPONENTS_VULNERABILITIES"
+                ON "COMPONENT"."ID" = "COMPONENTS_VULNERABILITIES"."COMPONENT_ID"
+             INNER JOIN "VULNERABILITY"
+                ON "COMPONENTS_VULNERABILITIES"."VULNERABILITY_ID" = "VULNERABILITY"."ID"
+             LEFT JOIN "EPSS"
+                ON "VULNERABILITY"."VULNID" = "EPSS"."CVE"
+             INNER JOIN "FINDINGATTRIBUTION"
+                ON "COMPONENT"."ID" = "FINDINGATTRIBUTION"."COMPONENT_ID"
+               AND "VULNERABILITY"."ID" = "FINDINGATTRIBUTION"."VULNERABILITY_ID"
+              LEFT JOIN "ANALYSIS"
+                ON "COMPONENT"."ID" = "ANALYSIS"."COMPONENT_ID"
+               AND "VULNERABILITY"."ID" = "ANALYSIS"."VULNERABILITY_ID"
+               AND "COMPONENT"."PROJECT_ID" = "ANALYSIS"."PROJECT_ID"
+             INNER JOIN "PROJECT"
+                ON "COMPONENT"."PROJECT_ID" = "PROJECT"."ID"
+             WHERE ${apiProjectAclCondition}
+             <#if !activeFilter>
+                AND "PROJECT"."INACTIVE_SINCE" IS NULL
+             </#if>
+             <#if !suppressedFilter>
+                AND "ANALYSIS"."SUPPRESSED" IS NULL OR NOT "ANALYSIS"."SUPPRESSED"
```

Comment:
Just realized this needs to be enclose in parentheses:

```suggestion
                AND ("ANALYSIS"."SUPPRESSED" IS NULL OR NOT "ANALYSIS"."SUPPRESSED")
```

Was wondering why filtering by `severity=critical` would still show findings with other severities. Without parentheses here, the entire `WHERE` condition would evaluate to true, if no analysis exists. 😅 

With this very minor thing fixed, everything else is working great!